src/api/schemas/extendedQueryNode.graphql

```diff
@@ -38,43 +87,22 @@ type Channel {
 
 type Category {
   id: ID!
-
   name: String!
-
   videos: [Video!]
 }
 
-type JoystreamMediaLocation {
-  dataObjectId: String!
-}
-
-type HttpMediaLocation {
-  url: String!
-}
-
-# In the future we can add IPFS, Dat, etc.
-union MediaLocation = JoystreamMediaLocation | HttpMediaLocation
-
-type KnownLicense {
-  code: String!
-  name: String
-  description: String
-  url: String
-}
-
-type UserDefinedLicense {
-  content: String!
-}
-
-union License = UserDefinedLicense | KnownLicense
-
-type LicenseEntity {
+type License {
   id: ID!
-  type: License!
+  # Should be Int?
+  code: String
+  url: String
   attribution: String
   videoLicense: [Video!]
+  customText: String
 }
-type VideoMedia {
+
+# TODO needs to be updated
```

Comment:
What's left to be done here? I think both `pixelWidth` and `pixelHeight` should be optional?